### PR TITLE
Improve alignment of icons on admin roles list

### PR DESCRIPTION
**Pull request by mjankowski**
Before

Screenshot 2024-09-28 at 17 19 24

After

Screenshot 2024-09-28 at 17 19 33

The other change keeps the icons with their text, eg...

Before

Screenshot 2024-09-28 at 17 29 23

After

Screenshot 2024-09-28 at 17 29 34

Good followup here would be to better align those as well.

### CODE REVIEW

File: app/javascript/styles/mastodon/admin.scss [hunk 1]
```diff
@@ -1054,6 +1054,10 @@ a.name-tag,
       }
     }
 
+    .icon {
+      padding-top: 5px;
+    }
```
Comment (ClearlyClaire) [inline app/javascript/styles/mastodon/admin.scss:1059]: The reason for the misalignment is `vertical-align: top`, addressing it with a `padding-top` feels especially weird.

File: app/javascript/styles/mastodon/tables.scss [hunk 2]
```diff
@@ -137,6 +137,7 @@ a.table-action-link {
   padding: 0 10px;
   color: $darker-text-color;
   font-weight: 500;
+  white-space: nowrap;
```
Comment (ClearlyClaire) [inline app/javascript/styles/mastodon/tables.scss:140]: Is this related?

File: app/javascript/styles/mastodon/admin.scss (hunk 1 above)
Comment (mjankowski) [reply app/javascript/styles/mastodon/admin.scss:1059]: Will look for alt method

Comment (mjankowski) [reply app/javascript/styles/mastodon/admin.scss]: Updated to use `vertical-align: middle` in this context, which will override the `top` from higher up in same file.

**Review by ClearlyClaire — APPROVED** [review]
